Documentation/Client.md

```diff
@@ -89,6 +89,20 @@ struct ImmunityFrame {
     double duration = 0;
     std::chrono::time_point<std::chrono::system_clock> timer;
 };
+
+struct SoundCreation {
+    SFXID ID;
+};
+
+struct SoundDestruction {
+    SFXID ID;
+};
+
+struct SoundDamage {
+    SFXID ID;
+};
+
```

Comment:
Maybe we could think about how to condense that into one Component, seems pretty inefficient doing it this way

Reply:
the SoundDamage component is currently on hold anyways, because it is hard to implement client side. so i instaed use the destruction of the bullets, maybe it should just get deleted completly.